src/buffer/out/search.h

```diff
@@ -53,27 +58,28 @@ class Search final
     std::pair<COORD, COORD> GetFoundLocation() const noexcept;
 
 private:
-    wchar_t _ApplySensitivity(const wchar_t wch) const;
+    wchar_t _ApplySensitivity(const wchar_t wch) const noexcept;
     bool Search::_FindNeedleInHaystackAt(const COORD pos, COORD& start, COORD& end) const;
-    bool _CompareChars(const std::wstring_view one, const std::wstring_view two) const;
+    bool _CompareChars(const std::wstring_view one, const std::wstring_view two) const noexcept;
     void _UpdateNextPosition();
 
     void _IncrementCoord(COORD& coord) const;
     void _DecrementCoord(COORD& coord) const;
 
-    static COORD s_GetInitialAnchor(const SCREEN_INFORMATION& screenInfo, const Direction dir);
+    COORD _GetInitialAnchor(Microsoft::Console::Types::IUiaData& uiaData, const Direction dir);

```

Comment:
I think it is not necessary to make this static though. It is called in the constructor but I think cpp does not require such a method to be static. Correct me if there are any issues here.

Reply:
This is less about the fact that it _could_ be static and more that it _should_ be. Since the function doesn't need any member variables, it doesn't really make sense for it to be a method, does it?

Reply:
changed